src/main/java/org/cascadebot/cascadebot/commands/music/StopCommand.java

```diff
@@ -0,0 +1,49 @@
+/*
+ *
+ *  * Copyright (c) 2019 CascadeBot. All rights reserved.
+ *  * Licensed under the MIT license.
+ *
+ */
+
+package org.cascadebot.cascadebot.commands.music;
+
+import net.dv8tion.jda.core.entities.Member;
+import org.cascadebot.cascadebot.commandmeta.CommandContext;
+import org.cascadebot.cascadebot.commandmeta.ICommandMain;
+import org.cascadebot.cascadebot.commandmeta.Module;
+import org.cascadebot.cascadebot.permissions.CascadePermission;
+
+public class StopCommand implements ICommandMain {
+
+    @Override
+    public void onCommand(Member sender, CommandContext context) {
+        if (context.getData().getMusicPlayer().getPlayer().getPlayingTrack() != null && !context.getData().getMusicPlayer().getPlayer().isPaused()) { // If the playing track isn't null, and the bot isn't paused
+            context.getData().getMusicPlayer().getPlayer().stopTrack(); // stop track
+            context.getData().getMusicPlayer().getTracks().clear(); // clear queue
+            context.replySuccess("Music has been stopped!");
+        } else if (context.getData().getMusicPlayer().getPlayer().getPlayingTrack() == null) { // If playing track is null
```

Comment:
don't do else if if you don't need to just do else

Reply:
got it